p2panda-rs/src/serde/hex_str.rs

```diff
@@ -5,7 +5,7 @@ use serde_bytes::{ByteBuf as SerdeByteBuf, Bytes as SerdeBytes};
 
 /// Helper method for `serde` to serialize bytes into a hex string when using a human readable
 /// encoding (JSON, GraphQL), otherwise it serializes the bytes directly (CBOR).
-pub fn serialize_hex<S>(value: &[u8], serializer: S) -> Result<S::Ok, S::Error>
+pub fn serialize_hex_bytes<S>(value: &[u8], serializer: S) -> Result<S::Ok, S::Error>
```

Comment:
Should we rename the deserialize method as well?

Reply:
Ah, I actually reverted this name change, thought I already pushed it but obviously not. Earlier I'd added a new util method and the names needed distinction. But I removed that and so this naming change isn't required anymore.